components/brave_wallet_ui/components/extension/sign-panel/sign-panel.stories.tsx

```diff
@@ -20,6 +20,7 @@ const signMessageData = {
   coin: BraveWallet.CoinType.ETH,
   isEip712: true,
   domainHash: '',
+  domain: '',
```

Comment:
Is this change intentional?

Reply:
Yes, I added it as a separate commit to try to distinguish it from the main change (though maybe I should have made a separate PR). Without the two fixes in that commit, hot-reloading of the storybook was failing with compilation errors.